.github/workflows/build.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       fail-fast: false
       matrix:
         platform: [ubuntu-18.04]
-        os_name: [linux_openresty, linux_tengine, linux_apisix_master_luarocks, linux_apisix_current_luarocks, linux_openresty_mtls]
+        os_name: [linux_openresty, linux_tengine, linux_apisix_master_luarocks, linux_apisix_current_luarocks, linux_openresty_mtls, linux_apisix_integrationtest]
```

Comment:
`linux_apisix_integrationtest`  we can remove the `linux_apisix_` prefix.

Reply:
how about `e2e_test`?

Reply:
that 's good idea! I will change it.